shuup_tests/api/test_basket_api.py

```diff
@@ -0,0 +1,207 @@
+# -*- coding: utf-8 -*-
```

Comment:
Yes, that's true. Currently the only proof I have of this is that the test suite was passing before I took care of the session details :D